### PR TITLE
Tolerations and affinities for monitoring workloads in QA.

### PR DESCRIPTION
**Pull request by emil2k**
Also disables prefix delegation in QA (deferred to later) and pins the AMI on the managed node group.

Ran HCLs through formatter.

### CODE REVIEW

File: test/integration/eu-west-1/k8s-qa/services/terragrunt.hcl [hunk 1]
```diff
@@ -144,35 +163,35 @@ inputs = {
...
   # Helm Exporter
   # --------------------------------------------------
 
-  helm_exporter_deploy              = "true"
-  helm_exporter_helm_chart_version  = "1.2.3"
-  helm_exporter_github_owner        = "dfds"
-  helm_exporter_repo_name           = "platform-manifests-qa"
-  helm_exporter_repo_branch         = "main"
+  helm_exporter_deploy             = "true"
+  helm_exporter_helm_chart_version = "1.2.3"
```
Comment (avnes) [inline test/integration/eu-west-1/k8s-qa/services/terragrunt.hcl:191]: We should attempt 1.2.4 that was released last week

**Review by avnes — CHANGES_REQUESTED** [review]

File: test/integration/eu-west-1/k8s-qa/services/terragrunt.hcl (hunk 1 above)
Comment (emil2k) [reply test/integration/eu-west-1/k8s-qa/services/terragrunt.hcl:191]: This is just a change in whitespace, but I can put up another PR to put this change in separately.

**Comment by emil2k** [comment]
@avnes I ran into an issue here with the Grafana Helm chart update hanging on "Multi-Attach Error" (https://github.com/grafana/helm-charts/issues/1184). This seems to be something new related to the persistence of the Grafana state (cc @samidbb). I will look into this as it breaks QA if I try to update the Grafana release.

**Comment by emil2k** [comment]
The issue above was due to the underlying pvs having required affinities to the original availability zone they were scheduled in by the EBS driver.

File: test/integration/eu-west-1/k8s-qa/services/terragrunt.hcl (hunk 1 above)
Comment (emil2k) [reply test/integration/eu-west-1/k8s-qa/services/terragrunt.hcl:191]: Extracted out to this PR: https://github.com/dfds/infrastructure-modules/pull/733

**Review by avnes — APPROVED** [review]

**Pull request merged**